docs/query-language.md

```diff
@@ -341,6 +341,14 @@ have the attribute specified by `field` with one of the values specified in
 
 **Result Type:** array of key/count objects
 
+**Caveats:**
+* TopN on a frame with ranked cache will return the top bitmaps sorted by count.
+* Non-ranked frames maintain a LRU (least recently used) cache, thus a TopN query on this frame will return bitmaps sorted in most recently setbit order.
+* The frame's cache size determines the limit of sorted bitmaps.  
+* The ranked cache maintains a sorted set of bitmaps by count up to the cache size.  
```

Comment:
It seems like this bullet point is redundant. Are you trying to convey something specific here?